217_contain_duplicates.py

```diff
@@ -0,0 +1,26 @@
+"""
+https://leetcode.com/problems/contains-duplicate/description/
+
+Given an array of integers, find if the array contains any duplicates. 
+Your function should return true if any value appears at least twice in the array, and it should return false if every element
+is distinct.
+"""
+class Solution(object):
+    def containsDuplicate(self, nums):
```

Comment:
LGTM

You could use your fav data structure `Set` here

Reply:
I feel the worse case scenario for my approach is O(n), for duplicates less than O(n) as it breaks before that. Wouldn't you have to do something similar for sets as well?

Reply:
You're right complexity wise it'll be identical it will just be 1 line with `Set` that is why I mentioned as it is slightly more pythonic to do it that way.

Reply:
Since this is already merged I think a more pythonic way to do this would be 

`return len(set(nums)) < len(nums)`